ConsoleApp6/reports/set1/MyBenchmarks.IntroIParam-report.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+<meta charset='utf-8' />
+<title>IntroIParam</title>
+
+<style type="text/css">
+	table { border-collapse: collapse; display: block; width: 100%; overflow: auto; }
+	td, th { padding: 6px 13px; border: 1px solid #ddd; }
+	tr { background-color: #fff; border-top: 1px solid #ccc; }
+	tr:nth-child(even) { background: #f8f8f8; }
+</style>
+</head>
+<body>
+<pre><code>
+BenchmarkDotNet=v0.10.11, OS=Windows 10 Redstone 3 [1709, Fall Creators Update] (10.0.16299.192)
+Processor=Intel Core i7-6650U CPU 2.20GHz (Skylake), ProcessorCount=4
+Frequency=2156248 Hz, Resolution=463.7685 ns, Timer=TSC
+.NET Core SDK=2.1.4
+  [Host]     : .NET Core 2.0.5 (Framework 4.6.26020.03), 64bit RyuJIT
+  DefaultJob : .NET Core 2.0.5 (Framework 4.6.26020.03), 64bit RyuJIT
+</code></pre>
+<pre><code></code></pre>
+
+<table>
+<thead><tr><th>Method</th><th>                                 Field</th><th>Mean</th><th>Error</th><th>StdDev</th><th>Median</th>
+</tr>
```

Comment:
Explanation:

(WithPercentageAsZombiesAtRandom 0,100,53):
- The setup will create a dictionary with 100 elements in it, and the initial capacity set is not set.
- It removes (100-53=)47 elements from the dictionary with no given bias as to which elements were removed, leaving 47 zombies in the dictionary.
- The benchmark method AddNew will add 53 new elements using newly modified dictionary
- The benchmark method AddOld will add 53 new elements using existing dictionary implementation



(WithPercentageAsZombiesAtRandom 0.5,100,52):
- The setup will create a dictionary with 100 elements in it, and the initial capacity set of (0.5*100=)50.
- It removes (100-52=)48 elements from the dictionary with no given bias as to which elements were removed, leaving 48 zombies in the dictionary.
- The benchmark method AddNew will add 52 new elements using newly modified dictionary
- The benchmark method AddOld will add 52 new elements using existing dictionary implementation